src/RedisPrefixTrait.php

```diff
@@ -94,4 +94,44 @@ protected function getPrefix() {
     return $this->prefix;
   }
 
+  /**
+   * From the given timestamp, with arbitrary increment as decimal, get
+   * the decimal value
+   *
+   * @param int|string $timestamp
+   *   "TIMESTAMP[.INCREMENT]" string
+   *
+   * @return string
+   *   "TIMESTAMP.INCREMENT" string.
+   */
+  public function getNextIncrement($timestamp = null) {
```

Comment:
Yeah, not sure I fully get this, some inline comments on what it is doing would be nice.